src/stcal/jump/jump.py

```diff
@@ -968,10 +970,10 @@ def compute_axes(expand_by_ratio, ellipse, expansion, jump_data):
     else:
         axis1 = ellipse[1][0] + expansion
         axis2 = ellipse[1][1] + expansion
-    axis1 = min(axis1, jump_data.max_extended_radius)
-    axis2 = min(axis2, jump_data.max_extended_radius)
+    axis1 = min(axis1 / 2, jump_data.max_extended_radius)
+    axis2 = min(axis2 / 2, jump_data.max_extended_radius)
 
-    return (round(axis1 / 2), round(axis2 / 2))
```

Comment:
Would you add a unit test verifying this fix?

Reply:
Added `test_ellipse_compute_axes_max_radius` in the test module. Without this patch, it fails as follows:

```
E           AssertionError: 
E           Arrays are not equal
E           
E           Mismatched elements: 1 / 2 (50%)
E           Max absolute difference among violations: 2
E           Max relative difference among violations: 0.5
E            ACTUAL: array([2, 2])
E            DESIRED: array([4, 2])
```

This is because it halved `max_extended_radius` on return, treating it as diameter instead.